### PR TITLE
[IMP] im_livechat: introduce PublicLivechatFeedbackView

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2928740

### CODE REVIEW

File: addons/im_livechat/static/src/models/public_livechat_feedback_view.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            this.messaging.livechatButtonView.chatWindow.legacyChatWindow.replaceContentWith(this.widget);
+            this.widget.on('feedback_sent', null, this.onFeedbackSend);
+            this.widget.on('send_message', null, this.onSendMessage);
+        },
+        _willDelete() {
+            this.widget.destroy();
+        },
+    },
+    recordMethods: {
+        onFeedbackSend(...args) {
```
Comment (oomsveta) [inline addons/im_livechat/static/src/models/public_livechat_feedback_view.js:28]: onFeedbackSen**t**?

File: addons/im_livechat/static/src/models/public_livechat_feedback_view.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            this.messaging.livechatButtonView.widget._closeChat(...args);
+        },
+        onSendMessage(...args) {
```
Comment (oomsveta) [inline addons/im_livechat/static/src/models/public_livechat_feedback_view.js:31]: how about having these methods private? (_ prefixed)

**Review by oomsveta — CHANGES_REQUESTED** [review]

**Comment by alexkuhn** [comment]
Changes done

File: addons/im_livechat/static/src/models/public_livechat_feedback_view.js (hunk 2 above)
Comment (alexkuhn) [reply addons/im_livechat/static/src/models/public_livechat_feedback_view.js]: IMO this guideline will drop sooner or later.
There's already no reason to keep it in models, and even framework JS stop doing it recently.
But I still made the change

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/96690

**Pull request closed without merging**